### ISSUE TITLE
HICO-DET download not working

### ISSUE THREAD

**Issue by PradKalkar** [zhihou7/HOI-CL#23]
When I run the first command in the readme - i.e. running the "download_dataset.sh" script; the HICO-DET dataset is not getting downloaded as a gzip file from the google drive link and thus tar command is not able to extract it. It seems there is some issue in lib/ult/Download_data.py file. Rather, when I inspected the contents, its HTML.

Please help.

**Comment by zhihou7** [comment]
Sorry for the removed url. Could you try this url? https://drive.google.com/file/d/1QZcJmGVlF9f4h-XLWe9Gkmnmj2z1gSnk/view

**Comment by PradKalkar** [comment]
Thanks for the reply. I tried the id which is provided in this url as well. But, its still fetching the file as an HTML.

**Comment by zhihou7** [comment]
I thought it might be the issue from the update in google drive. I recently can’t not download file with gdown. I can download the file on safari. Could u download from the website?

**Comment by PradKalkar** [comment]
Yes, I can download from the website but not from the download_dataset.sh script.

**Comment by zhihou7** [comment]
Hi @PradKalkar, Thanks for your comment. There is indeed some issue in ``lib/ult/Download_data.py`` file. I have switch to use gdown to download the files, and update the code.

Many Thanks to you!

Regards,

**Comment by PradKalkar** [comment]
Thanks for the quick response @zhihou7 :)
